app/pages/list/list.js

```diff
@@ -0,0 +1,34 @@
+import {IonicApp, Page, NavController, NavParams} from 'ionic/ionic';
+
+@Page({
+  templateUrl: 'build/pages/list/list.html'
+})
+export class ListPage {
+  constructor(app: IonicApp, nav: NavController, navParams: NavParams) {
```

Comment:
This view introduces a new component called NavParams which is added through the constructor. This will allow you to return the details of a view that has been navigated to